storage/remote/codec_test.go

```diff
@@ -43,12 +43,12 @@ var (
 		Schema:          2,
 		ZeroThreshold:   1e-128,
 		ZeroCount:       0,
-		Count:           0,
+		Count:           3,
```

Comment:
note to reviewers: this is not related to NHCB. just noticed that the histogram was not valid. Count wrong and negative count in a bucket later.